charts/grafana/ci/with-sidecars-envvaluefrom-values.yaml

```diff
@@ -0,0 +1,38 @@
+extraObjects:
+  - apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: '{{ include "grafana.fullname" . }}'
+    data:
+      var1: "value1"
+  - apiVersion: v1
+    kind: Secret
+    metadata:
+      name: '{{ include "grafana.fullname" . }}'
```

Comment:
CI says a secret with that name already exists.

Could you add a suffix here like test? please update the references in envValueFrom

Reply:
@jkroepke done.